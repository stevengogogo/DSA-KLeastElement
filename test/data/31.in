10 50000
5224 8876 3489 -3351 -5444 -9862 -7018 2440 -1373 -3328
Delete 3
Insert 7 3531
Reverse 10 10
Insert 5 -7542
Query 3 5 1
Reverse 4 7
Reverse 2 7
Delete 10
Reverse 2 6
Insert 11 -7152
Reverse 8 8
Delete 10
Insert 9 -2908
Delete 11
Query 2 5 4
Query 9 10 1
Reverse 3 4
Reverse 4 4
Query 6 10 3
Delete 7
Reverse 3 5
Query 4 6 1
Insert 9 -7511
Insert 7 -4305
Reverse 5 5
Reverse 7 8
Reverse 5 11
Insert 2 8206
Reverse 2 4
Reverse 8 10
Delete 8
Query 3 4 1
Query 7 8 1
Query 10 10 1
Insert 3 -3581
Delete 12
Insert 2 -9185
Delete 4
Query 8 11 4
Reverse 3 4
Delete 2
Query 1 8 7
Insert 11 -7667
Reverse 3 9
Reverse 1 1
Query 11 11 1
Insert 12 -7374
Query 2 8 6
Query 8 9 2
Query 12 12 1
Reverse 11 11
Delete 8
Insert 10 2199
Query 12 12 1
Delete 5
Reverse 1 3
Query 8 11 2
Insert 5 -7426
Delete 3
Query 5 11 5
Delete 8
Query 6 8 3
Reverse 2 7
Delete 7
Reverse 7 7
Delete 8
Reverse 3 7
Delete 7
Delete 3
Query 2 4 1
Insert 7 6535
Delete 6
Reverse 3 6
Query 4 5 1
Query 5 6 2
Query 2 3 1
Query 5 5 1
Delete 2
Reverse 2 2
Delete 3
Insert 4 7513
Delete 3
Delete 4
Delete 3
Query 2 2 1
Insert 1 -6170
Query 1 2 2
Insert 3 -5202
Reverse 3 4
Reverse 1 3
Query 3 4 2
Query 3 3 1
Query 4 4 1
Query 1 4 3
Query 4 4 1
Reverse 2 3
Query 1 1 1
Insert 3 -3092
Reverse 2 4
Reverse 5 5
Query 5 5 1
Reverse 5 5
Insert 2 7631
Insert 1 -387
Insert 5 -4100
Reverse 7 8
Query 3 6 2
Query 4 8 4
Insert 1 -9126
Delete 1
Reverse 7 8
Reverse 4 8
Query 3 8 6
Reverse 8 8
Query 4 7 2
Reverse 6 6
Reverse 7 8
Insert 3 -9553
Delete 9
Delete 8
Delete 1
Delete 2
Delete 1
Reverse 1 4
Query 1 2 2
Insert 2 4459
Delete 1
Insert 2 -6751
Insert 2 7243
Query 3 4 2
Query 6 6 1
Query 5 6 1
Delete 5
Insert 5 -3661
Reverse 5 6
Query 2 6 3
Reverse 3 4
Reverse 1 6
Query 1 4 4
Delete 1
Insert 5 5860
Query 5 5 1
Insert 3 8369
Query 1 3 2
Reverse 1 2
Query 4 5 1
Query 5 7 1
Query 4 5 2
Insert 5 -2342
Delete 2
Reverse 6 7
Query 4 5 2
Query 4 4 1
Delete 3
Reverse 5 6
Reverse 5 5
Query 4 5 2
Delete 5
Reverse 3 5
Insert 3 8245
Query 2 3 1
Delete 3
Insert 2 -3660
Delete 2
Query 2 3 1
Query 4 5 1
Insert 4 -3040
Reverse 1 6
Insert 5 -3357
Query 3 7 4
Insert 3 -5487
Delete 8
Reverse 2 4
Insert 6 -7639
Insert 6 -2007
Reverse 6 8
Delete 6
Query 8 8 1
Delete 3
Query 3 3 1
Reverse 5 5
Query 1 4 4
Delete 3
Reverse 4 6
Delete 6
Reverse 5 5
Query 5 5 1
Delete 4
Insert 4 -4521
Delete 2
Delete 4
Delete 2
Reverse 1 1
Query 1 2 1
Insert 1 153
Delete 2
Query 2 2 1
Query 2 2 1
Insert 2 7290
Query 2 3 1
Delete 2
Delete 2
Query 1 1 1
Insert 2 -4560
Delete 2
Query 1 1 1
Delete 1
Insert 1 8037
Delete 1
