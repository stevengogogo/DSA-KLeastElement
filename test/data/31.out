-7542
-3351
-2908
2440
-9862
-3351
-7511
-5444
-2908
5224
-7667
2440
8206
-7374
-7374
-7374
-5444
2199
-7542
-7426
-4305
-7542
-7426
6535
-2908
-5202
-6170
-5202
-2908
-5202
6535
-5202
-3092
-3092
7631
-5202
-3092
-6170
7631
-5202
-3661
7631
5860
7631
-6170
4459
7243
7243
-2342
7243
8245
4459
-2342
4459
8369
7243
7243
-2007
-4521
-4521
-4521
-4521
153
153
